### PR TITLE
Make Smores

### PR DESCRIPTION
**Pull request by williamfbaur**
Create fire.md with 2 statements.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2674)

### CODE REVIEW

File: english/eating-walls/light-fire/fire.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```
Comment (casey-collab) [inline english/eating-walls/light-fire/fire.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/eating-walls/light-fire/fire.md (hunk 1 above)
Comment (williamfbaur) [inline english/eating-walls/light-fire/fire.md]: I can't seem to find where to go to start the story.  Where do I go to start language.md?
Thank you.
William F. Baur

Comment (williamfbaur) [inline english/eating-walls/light-fire/fire.md]: I can't seem to find where to go to start the story. Where do I go to start
language.md?
Thank you.
William F. Baur

On Fri, Oct 30, 2015 at 7:35 PM, Casey Collaborator <
notifications@github.com> wrote:

> In english/eating-walls/light-fire/fire.md
> <https://github.com/udacity/create-your-own-adventure/pull/2674#discussion_r43562811>
> :
>
> > @@ -0,0 +1,4 @@
> > +You build a camp-fire, grab a fistful of the wall and put it on a stick.
>
> It looks like this page is not reachable from the rest of the story. That
> is, if a reader starts in language.md and keeps clicking links, they will
> never find this page. You can fix this by finding a good place in the
> existing story to add an option with a link that leads to this page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/2674/files#r43562811>
> .
>



-- 
Sincerely,
William F. Baur

Comment (cbuckey-uda) [inline english/eating-walls/light-fire/fire.md]: @williamfbaur, what you should do is pick any existing page of the story, and add an option, e.g. "make smores" with a link to your new page. Then when the user clicks that option, they will be taken to your page.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**